src/MusicStore/Areas/Admin/Controllers/StoreManagerController.cs

```diff
@@ -130,19 +146,19 @@ public IActionResult Create()
         // POST: /StoreManager/Edit/5
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Edit(Album album)
+        public async Task<IActionResult> Edit(CancellationToken requestAborted, Album album)
```

Comment:
i wish the actual model binding object first in the parameter list. Probably make cancellation token last but one if there are default parameters?